webapp/messages/en.json

```
@@ -300,24 +300,30 @@
       "btc-deposit-come-back-delay-note": "If you do not receive your deposit after {hours} hours return to this page to manually claim.",
       "deposit-completed": "Deposit completed",
       "click-to-confirm": "Click on 'Confirm Deposit Manually' to deposit your Bitcoin to Hemi.",
-      "confirm-deposit": "Confirm deposit",
+      "confirm-deposit-on": "Confirm deposit on {networkName}",
       "deposit-finalized": "Deposit finalized",
       "deposit-on-its-way": "Your deposit is on its way! Please wait while the blockchain finalizes the transaction.",
-      "get-your-funds-on-hemi": "Get your funds on Hemi",
+      "get-your-funds-on-hemi": "Get your funds on {networkName}",
```

I believe now this locale key needs to be updated to match the description

```suggestion
      "get-your-funds-on-network": "Get your funds on {networkName}",
```


Also acceptable

```suggestion
      "get-your-funds-on": "Get your funds on {networkName}",
```

---

Done. Thanks!